doc/man3/X509_NAME_print_ex.pod

```diff
@@ -35,10 +35,11 @@ characters. Multiple lines are used if the output (including indent) exceeds
 
 =head1 NOTES
 
-The functions X509_NAME_oneline() and X509_NAME_print() are legacy functions which
+The functions X509_NAME_oneline() and X509_NAME_print()
 produce a non standard output form, they don't handle multi character fields and
-have various quirks and inconsistencies. Their use is strongly discouraged in new
-applications.
+have various quirks and inconsistencies.
+Their use is strongly discouraged in new applications and they will
+be deprecated in a future release.
```

Comment:
I am in favor of that, but this PR is just for documentation.  I left us "wiggle room" to do it whenever we want.  In fact I changed it to say "could be deprecated"